core/modules/system/system.install

```diff
@@ -3343,6 +3343,22 @@ function system_update_1085() {
   }
 }
 
+/**
+ * Enable static caching of the "system.date" configuration file.
+ */
+function system_update_1086() {
+  $config = config('system.date');
+  $data = $config->get();
+  // Put the _config_static option at the top of the file.
+  unset($data['_config_name']);
+  $data = array_merge(array(
+    '_config_name' => 'system.date',
+    '_config_static' => TRUE,
+  ), $data);
```

Comment:
Wouldn't `array_splice()` be a cleaner way to do this here, instead of unsetting the first item and adding it back? Something like this perhaps? (untested):

```suggestion
  // Put the "_config_static" option at the top of the file, right after "_config_name".
  $static_cache = array('_config_static' => TRUE);
  array_splice($data, 1, 0, $static_cache);
```

Reply:
...hmm, `array_splice()` won't play that nice with associative arrays 🤔 ...see: https://stackoverflow.com/questions/1783089/array-splice-for-associative-arrays

So you'd need to do something like this instead:

```suggestion
  $new_data = array_slice($data, 0, 1, TRUE) + array('_config_static' => TRUE) + array_slice($data, 1, NULL, TRUE);
```

...which seems equally messy/confusing.

Reply:
Yeah, assigning items to the middle of an array is often tricky. I think leaving this as-is might be the easiest solution.